components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -180,6 +182,45 @@ Get ancestor organization ids (including itself) of a given organization. The li
     }
 
     @Override
+    public void clearBrandingResolverCacheHierarchy(String currentTenantDomain) throws BrandingPreferenceMgtException {
+        String organizationId = getOrganizationId();
+
+        try {
+            OrganizationManager organizationManager =
```

Comment:
L189 can be moved out of try block. 
after that  you can getOrganizationId() and do the null check.

Reply:
Resolved by https://github.com/wso2-extensions/identity-branding-preference-management/pull/17/commits/feb429029efca6e212d803950f40a928eb48931d